tests/settings/ConfigurationGatheringTest.py

```diff
@@ -228,7 +229,8 @@ def test_back_saving(self):
         with open(filename, 'r') as f:
             lines = f.readlines()
         self.assertEqual(['[cli]\n',
-                          'config = some_bad_filename\n'], lines)
+                          'config = some_bad_filename\n',
+                          'find_config = True\n'], lines)
```

Comment:
as `find_config` defaults to True, there is no need to include it in every section, not even in `cli` section.